src/m3ninx/index/segment/fst/README.md

```diff
@@ -1,6 +1,58 @@
 FS Segment
 ===========
 
+- Version 1.1: Adds support for a metadata proto object per Field. This is used to
```

Comment:
I kind of just realized, is there really no checksums for any of our index files? If not, I think that's something we're going to have to work on rather soon.

Let me know if there is, AFAIK I cannot recall there being any.

Reply:
Yep, no checksumming internally within the m3ninx side, we rely on the checksumming by virtue of the digest files we create in the `dbnode/fs/persist.indexWriter`.

Reply:
Oh yeah, ok yeah we do actually checksum the segments when we read them from disk first time.  

It's just if they decay over time, we don't validate them everytime we read data (like we do when we look up data chunks for time series data with the retriever/seeker). So basically we only validate once at boot up time.

Reply:
Yeah pretty much. We can add this with a v1.2 without much issue.